### ISSUE TITLE
Notification on / off via Command line? Bash script?

### ISSUE THREAD

**Issue by chrissierra** [ccrisan/motioneye#659]
Hello, 
there is a way for enable and disabled notification on motion via command line?

**Comment by chrisf4lc0n** [comment]
Nope, not at the moment it is not as whatever you will put in it will get overwritten by MotionEye UI.

**Comment by chrissierra** [comment]
Ok, only you can do this via interface web?

**Comment by chrisf4lc0n** [comment]
Pretty much yes.

**Comment by feelthepayne** [comment]
I do this with alternative config files - one with notification enabled, the other with notification turned off.  A cron job checks the arp cache on the router for the MAC address of my mobile and updates the symlink of the config file accordingly and reloads the motioneye process.

next step when I get 5 minutes is to use geofencing with ifttt and an AWS Lambda function of some sort as the ARP cache on the router can be unreliable particularly if i've turned my WiFi off for what ever reason.

**Comment by chrisf4lc0n** [comment]
@feelthepayne wow! I have not thought about that. I tried it with sed command and it worked until I started the MotionEye GUI.
Really great idea mate, kudos to you!
